packages/cli/src/commands/pg/backups/capture.ts

```diff
@@ -19,7 +19,7 @@ export default class Capture extends Command {
   };
 
   static args = {
-    database: Args.string(),
+    database: Args.string({description: 'The config var exposed to the owning app containing the database configuration.'}),
```

Comment:
This is very confusing. What does `DATABASE` actually mean here? Can you give an example value? Is it something like `postgresql-animate-91581` as in the database name?

Reply:
It's going to be one of these typically:
<img width="608" alt="image" src="https://github.com/user-attachments/assets/bf00dbfc-b774-40b5-80ef-d9772e4b3d9e">

Reply:
It's going to be one of these typically:
<img width="244" alt="image" src="https://github.com/user-attachments/assets/9ecf834d-24dc-4ed5-be4e-17ab350a84e6">